### PR TITLE
cmake: remove jsoncpp dependency

### PR DESCRIPTION
**Pull request by tswhison**
jsoncpp is no longer used and is not present on some build systems,
causing the configure step to fail. Removes dependency on jsoncpp in
favor of json-c.

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -88,11 +88,10 @@ include_directories(
   ${OPAE_SDK_SOURCE}/tests)
 
 add_library(commonlib SHARED common_test.h common_test.cpp)
-target_link_libraries(commonlib opae-c ${GTEST_BOTH_LIBRARIES} ${jsoncpp_LIBRARIES})
+target_link_libraries(commonlib opae-c ${GTEST_BOTH_LIBRARIES})
```
Comment (r-rojo) [inline tests/CMakeLists.txt]: Does commonlib need to link with json-c?

Comment (tswhison) [reply tests/CMakeLists.txt]: Good call, I think so.

**Comment by r-rojo** [comment]
We should probably remove jsoncpp from .travis.yml while we're at it 😄

**Comment by tswhison** [comment]
@r-rojo Removed it from .travis.yml.

**Pull request merged**